gulp/copy.js

```diff
@@ -7,23 +7,20 @@ module.exports = (blueprint, gulp, plugins) => {
     var mergeStream = require("merge-stream");
     var path = require("path");
 
-    blueprint.task("copy", "files", [], (project) => {
-        // allow for no-op on project dependencies
-        if (project.copy === false) {
-            return;
-        }
```

Comment:
`blueprint.task` always generated a root task that depended on corresponding tasks in other projects. so `copy-files-docs` depended on (the existence of) `copy-files-core`, which is why we had to add the no-op shortcut. this was awkward because there wasn't actually a dependency between the copy tasks (like, they're completely separate operations), whereas there _is_ a real dependency for the typescript tasks (core must be compiled before datetime).

in this new world, the `copy` task simply ignores the `depTaskNames` argument and voila, packages can be copied without any dependencies, as they should be!